content/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Deny_List.adoc

```diff
@@ -0,0 +1,127 @@
+== Deny list implementation guidance
+
+A deny list is a curated list of authenticators that should **not** be allowed to register and authenticate to your application. Any authenticator that is not part of your curated list should be allowed to register. This scenario works well for applications that want a degree of control to prevent vulnerable authenticators from being used in the application. The guidance given on this page will teach you how to implement a deny list in your application, that will limit certain authenticators from registering.
+
+This scenario works best when:
+
+* Your organization has a security organization actively monitoring for authenticator issues
+* You want to prevent users from registering authenticators like copyable passkeys
+* Your organization has decided not to trust certain authenticator manufacturers
+* You want your application to automatically remediate rather than send user alerts for authenticators with vulnerabilities
+
+
+=== User experience
+
+Figure 1 demonstrates a sample user experience for a registration process that implements an deny list
+
+image::../images/denylist1.jpg[]
+**Figure 1**
+
+In this flow the happy path comes from a user who immediately opts into sending attestation data, and ensuring that they are using an authenticator that is not part of your curated deny list.
+
+It’s worth noting that in this experience most of the logic/decision points are left up to the application, with little intervention from the user, aside from creating the initial registration, and reacting to any request rejections.
+
+[NOTE]
+====
+One major decision point you need to make is whether or not to accept untrusted attestation. The flow above assumes that a user is sending an attestation statement along with their registration, and that it belongs to a trusted authenticator. If your application is aiming to be highly permissive then it may not always be beneficial to require your user to send an attestation statement. If you don’t require attestation it’s possible that a user registers with a vulnerable device, but it’s not detected due to the attestation statement not being sent.
+
+If the deny list is for a consumer facing/low assurance application then it might be fine to allow users to opt out of the need to send attestation.
+
+If the deny list is for a high risk application then you should always require the user to send attestation in order to establish the make/model of the device. This will also work well for scenarios where you want to prevent copyable passkeys from being used in your application
```

Comment:
Consider rewording 'a high risk application' to 'an application that requires a high level of authentication assurance'